scipy/special/_add_newdocs.py

```diff
@@ -2391,6 +2391,67 @@ def add_newdoc(name, doc):
            integrals," Numer. Algorithm, vol. 10, no. 1, pp. 13-26, 1995.
            https://arxiv.org/abs/math/9409227
            https://doi.org/10.1007/BF02198293
+
+    Examples
+    --------
+    Basic homogeneity property:
+
+    >>> from scipy.special import elliprc
+    >>> from numpy import sqrt
```

Comment:
In all examples (in the background) we have `import numpy as np`. So no import needed and you can directly use `np.xxx`. Please adapt here and elsewhere.

Reply:
Thanks @tupui. Addressed in f0f5da8 and passed local refguide-check. Please mark as resolved if you're happy! I will eventually squash.